### PR TITLE
Folder Restructure and Image Decompressing

### PR DESCRIPTION
**Pull request by shantanuparab-tr**
1. **Folder Restructure** 
The act module is not detected when scripts are placed in the /scripts folder. Moved the script to /act folder and updated the import statements accordingly.
2. **Decompress Images** 
The images recorded in the .hdf5 episodes are in compressed format, this causes issue with tensor size mismatch in torch.einsum() method. 
Added a compressed flag which is extracted from the .hdf5 file to decompress the images when required.
3. **Auto Retrieval of number of episodes**
Number of episodes was required as a config parameter. Fixed the load_data method to locate and search for the episode files and count them to be used as num_episodes.
4. **Removed the old scripts folder**
Deleted the old script folder that was unnecessary.

### CODE REVIEW

File: act/scripts/imitate_episodes.py
Comment (lukeschmitt-tr) [inline act/scripts/imitate_episodes.py:1]: Make sure to remove the old script file

**Review by lukeschmitt-tr — CHANGES_REQUESTED** [review]

**Comment by shantanuparab-tr** [comment]
Removed the old scripts folder

File: act/scripts/imitate_episodes.py
Comment (shantanuparab-tr) [reply act/scripts/imitate_episodes.py:1]: Removed the old scripts folder

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@ fuse.cfg
 # Generation results
 results/
 
+
```
Comment (lukeschmitt-tr) [inline .gitignore]:
```suggestion
```

File: act/imitate_episodes.py [hunk 2]
```diff
@@ -472,5 +472,8 @@ def plot_history(train_history, validation_history, num_epochs, ckpt_dir, seed):
     parser.add_argument('--hidden_dim', action='store', type=int, help='hidden_dim', required=False)
     parser.add_argument('--dim_feedforward', action='store', type=int, help='dim_feedforward', required=False)
     parser.add_argument('--temporal_agg', action='store_true')
+    argument = vars(parser.parse_args())
 
+    # print(f"Training X, Checkpoint: {argument['ckpt_dir']} ")
+    # sys.exit()
```
Comment (lukeschmitt-tr) [inline act/imitate_episodes.py]:
```suggestion
```

**Review by lukeschmitt-tr — CHANGES_REQUESTED** [review]

**Review by lukeschmitt-tr — APPROVED** [review]

**Pull request merged**